test/integration_test/basic_dmthin_test.go

```diff
@@ -332,6 +354,15 @@ func BasicInstallDmthin(tc *types.TestCase) func(*testing.T) {
 	}
 }
 
+func BasicInstallDmthinFailed(tc *types.TestCase) func(*testing.T) {
```

Comment:
Put a comment here that this test will be used to test cases where the setup will not satisfy DMthin requirements like 
- < 8 core CPU
- < 8GB RAM
- <4.18 kernel version

Reply:
Done.